src/NServiceBus.Core.Analyzer/Sagas/RewriteConfigureHowToFindSagaFixer.cs

```diff
@@ -0,0 +1,194 @@
+                var mapping = mappings[i];
+
+                mapSagaInvocation = InvocationExpression(
+                    MemberAccessExpression(
+                        SyntaxKind.SimpleMemberAccessExpression,
+                        mapSagaInvocation.WithTrailingTrivia(EndOfLine(NewLine)),
+                        Token(SyntaxKind.DotToken).WithLeadingTrivia(Indent(2)),
+                        CreateGenericMappingMethod(mapping.IsHeader ? "ToMessageHeader" : "ToMessage", mapping.MessageTypeSyntax.ToFullString())
+                    )
+                )
+                .WithArgumentList(
+                    ArgumentList(
+                        SingletonSeparatedList(mapping.MessageMappingExpression)
+                    )
+                );
+            }
+
+            var block = Block(
+                Token(SyntaxKind.OpenBraceToken).WithLeadingTrivia(Indent()).WithTrailingTrivia(EndOfLine(NewLine)),
+                SingletonList<StatementSyntax>(
+                    ExpressionStatement(mapSagaInvocation, Token(SyntaxKind.SemicolonToken).WithTrailingTrivia(EndOfLine(NewLine)))
+                ),
+                Token(SyntaxKind.CloseBraceToken).WithLeadingTrivia(Indent())
+            );
+
+            // For debugging only
+            //System.Console.WriteLine("====== Block Full String =======");
+            //System.Console.WriteLine(block.ToFullString());
+            //System.Console.WriteLine("================================");
+
+            var newMethod = MethodDeclaration(
+                    PredefinedType(Token(SyntaxKind.VoidKeyword)),
+                    Identifier("ConfigureHowToFindSaga"))
+                .WithModifiers(
+                    TokenList(
+                        new[]{
+                            Token(SyntaxKind.ProtectedKeyword),
+                            Token(SyntaxKind.OverrideKeyword)}))
+                .WithParameterList(method.ParameterList.NormalizeWhitespace())
+                .NormalizeWhitespace().WithLeadingTrivia(Whitespace(baseIndentText)).WithTrailingTrivia(EndOfLine(NewLine))
+                .WithBody(block)
+                .WithExpressionBody(null)
+                .WithTrailingTrivia(method.GetTrailingTrivia());
+
+            var root = await document.GetSyntaxRootAsync(cancellationToken).ConfigureAwait(false);
+            var newRoot = root.ReplaceNode(method, newMethod);
+            var newDocument = document.WithSyntaxRoot(newRoot);
+
+            return newDocument;
+        }
+
+        static GenericNameSyntax CreateGenericMappingMethod(string methodName, string messageType)
+        {
+            return GenericName(methodName)
+                .WithTypeArgumentList(TypeArgumentList(SingletonSeparatedList<TypeSyntax>(IdentifierName(messageType))));
+        }
+
+        static ArgumentListSyntax CreateCorrelationIdMapping(string correlationId)
+        {
+            return ArgumentList(
+                //Token(SyntaxKind.OpenParenToken),
```

Comment:
Leaving this comment for debugging only purposes too?

Reply:
Nope, vestigial from when I was fighting the whitespace dragon.
```suggestion
```